packages/portal/src/composables/itemMediaPresentation.js

```diff
@@ -11,6 +11,14 @@ const activeAnnotation = ref(null);
 const page = ref(1);
 const presentation = ref(null);
 
+const annotationAtCoordinate  = (coordinate, fullExtent) => {
+  const coordinateToCompare = [coordinate[0], fullExtent[3] - coordinate[1]];
```

Comment:
this could happen as part of the handleMapClick function in the MeidaImageViewer component.